site/.markdownlint.json

```diff
@@ -0,0 +1,11 @@
+{
+  "line-length": false,
+  "no-inline-html": false,
```

Comment:
Do we need inline HTML?

Reply:
You are using some Vue components and divs on the home page. You can't get rid of them, so I just disabled this rule.

Reply:
Those are exceptions, I think we generally don't want HTML (since we now no longer need code groups with HTML). So it's better to add linting exception comments to these places, and have the rule enabled. What do you think?